uninstall/uninstall_linux.go

```diff
@@ -0,0 +1,90 @@
+package uninstall
+
+import (
+	"time"
+	"io/ioutil"
+	"os/exec"
+	"github.com/pufferpanel/pufferd/logging"
+	"syscall"
+)
+
+
+func StartProcess(){
+	killDaemon()
+	deleteFiles()
+	deleteUser()
+}
+
+func killDaemon(){
+	exec.Command("killall", "-15" "-u", "pufferd").Run()

```

Comment:
The kill -15 should act like a systemctl <stop> stop...
I would leave it, instead, I could add the systemctl, what do you think?